content/docs/fsm_efsm.md

```diff
@@ -10,8 +10,11 @@ GraphWalker uses 2 different state machines. The **finite** and the **extended f
 The difference is that the EFSM has the capabilities to add "if-statements" for the edges.
 
 ## FSM - Finite State Machine
-If you can design your test using FSM only, we strongly  encourage you to do so. The models will be much simpler to understand, thus easier to maintain.
+If you can design your test using FSM only, we strongly  encourage you to do so. 
```

Comment:
I have transferred some sentences on new lines due to some lines are very long and it is not readable in IDE.
It requires horizontal scrolling and anybody who read it may lost focus on which line he was.